src/main/java/org/jabref/gui/entryeditor/CommentsTab.java

```diff
@@ -80,14 +80,14 @@ protected SequencedSet<Field> determineFieldsToShow(BibEntry entry) {
         comments.add(StandardField.COMMENT);
 
         // Also show comment field of the current user (if enabled in the preferences)
-        if (entry.hasField(userSpecificCommentField) || entryEditorPreferences.shouldShowUserCommentsFields()) {
```

Comment:
Can you please explain these changes?

Reply:
If an entry has a `userSpecificCommentField` but the `show user comments field` option is disabled in the preferences, it should not be visible

Reply:
It should display existing fields. Only the empty field should not be shown. Therefore the `||`.

Reply:
Got it, I will redo this change.